cNVMe/Controller.cpp

```diff
@@ -189,11 +189,7 @@ namespace cnvme
 			COMPLETION_QUEUE_ENTRY completionQueueEntryToPost = { 0 };
 			UINT_32 memoryPageSize = ControllerRegisters->getMemoryPageSize();
 
-			if (memoryPageSize == 0)
```

Comment:
I want to assert here if this fails... may as well die. We can go over cNVMe's asserting if anyone has any questions.

Reply:
(not directly related)